README.adoc

```diff
@@ -1,26 +1,33 @@
 = gitbucket-asciidoctor-plugin
-:plugin-version: 0.1.0
+:plugin-version: 0.1.2
 
-image:https://travis-ci.org/lefou/gitbucket-asciidoctor-plugin.svg["Build Status", link="https://travis-ci.org/lefou/gitbucket-asciidoctor-plugin"]
+//image:https://travis-ci.org/lefou/gitbucket-asciidoctor-plugin.svg["Build Status", link="https://travis-ci.org/lefou/gitbucket-asciidoctor-plugin"]
 
-A https://github.com/takezoe/gitbucket[GitBucket] plug-in that provides AsciiDoc rendering capabilities.
+A https://github.com/gitbucket/gitbucket[GitBucket] plug-in that provides AsciiDoc rendering capabilities.
 All `.adoc` (and also the deprecated `.asciidoc` and `.ad` suffix) files will be automatically rendered as AsciiDoc, supporting most features of the http://asciidoctor.org/[Asciidoctor project], including rendering of a table of contents.
 
 == Download
 
-You can download a precompiled version from the https://github.com/lefou/gitbucket-asciidoctor-plugin/releases[Release page].
+You can download a precompiled version from the https://github.com/nobusugi246/gitbucket-asciidoctor-plugin/releases[Release page].
```

Comment:
I would like to have the released artifacts at the same place as the code. Could you please remove this change?